content/applications/inventory_and_mrp/quality/quality_management/failure_locations.rst

```diff
@@ -0,0 +1,107 @@
+--> Control Points`. Select an existing |QCP| from the list, or create a new one by clicking
+:guilabel:`New`.
+
+.. note::
+   The following instructions only detail the configuration settings necessary for adding a failure
+   location to a |QCP|. For a full overview of |QCPs| and all of the options available when
+   configuring them, see the documentation on :doc:`quality control points
+   <quality_control_points>`.
+
+In the :guilabel:`Control Per` field on the |QCP| form, select the :guilabel:`Quantity` option.
+Doing so causes a :guilabel:`Failure Locations` field to appear on the form. This field is only
+available when the :guilabel:`Quantity` option is selected.
+
+In the :guilabel:`Failure Locations` field, select one or more locations from the drop-down menu. To
+create a new location, type the desired location name into the field, and then select
+:guilabel:`Create "[name]"` from the drop-down menu.
+
+.. image:: failure_locations/qcp-form.png
+   :align: center
+   :alt: A QCP form in the Quality app, configured with a failure location.
+
+Send products to failure location
+=================================
+
+Once a |QCP| has been configured with one or more failure locations, products that fail a check
+created by the |QCP| can be routed to one of the locations.
+
+To do so, open an order that requires a quality check created by a |QCP| configured with a failure
+location. For example, navigate to :menuselection:`Inventory app --> Operations --> Receipts`, and
+select a receipt.
+
+At the top of the selected order, click the :guilabel:`Quality Checks` button to open a pop-up
+window, from which the quality check can be processed. At the bottom of the pop-up window, click the
+:guilabel:`Fail` button to fail the quality check, and open a second pop-up window, titled
+:guilabel:`Quality Check Failed for [Product]`.
+
+In the :guilabel:`Quantity Failed` field, enter the quantity of the product that failed to pass the
+quality check. In the :guilabel:`Failure Location` field, select a location to which the failed
+quantity should be sent. Then, click :guilabel:`Confirm` at the bottom of the pop-up window to close
+it.
+
+.. image:: failure_locations/failed-pop-up.png
+   :align: center
+   :alt: The pop-up window that appears after a quality check fails.
+
+Finally, on the order, click the :guilabel:`Validate` button at the top of the page. Doing so
+confirms that the products that failed the quality check were sent to the failure location,while
+products that passed it were sent to their normal storage locations.
+
+View failure location inventory
+===============================
+
+To view the product quantities stored in a failure location, navigate to :menuselection:`Inventory
+app --> Configuration --> Locations`. Select a failure location from the list.
+
+A failure location's page lists all of the products stored within the location, along with the
+quantity of each.
```

Comment:
Doesn't this only appear when the 'Current Stock' smart button at the top of a location form is clicked? If so, that might be worth clarifying here. Because, according to the runbot I'm using for this review, selecting a failure location from the list, only takes me to that location's form page -- from which I can access the 'Current Stock' smart button. Either way, it's probably best to double-check this during your revisions. Thanks!